api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -59,4 +60,7 @@ message IPTagging {
   // Tracked by issue https://github.com/envoyproxy/envoy/issues/2695]
   // The set of IP tags for the filter.
   repeated IPTag ip_tags = 4 [(validate.rules).repeated = {min_items: 1}];
+
+  // Custom header to emit tag.
+  string custom_header = 3;
```

Comment:
could we reuse unallocated field index or should pick the next available one, e.g., use `5` instead of `3`

Reply:
> could we reuse unallocated field index or should pick the next available one, e.g., use `5` instead of `3`

seems better to pick the next available one